packages/components/src/form-token-field/stories/index.tsx

```diff
@@ -123,3 +123,11 @@ WithCustomRenderItem.args = {
 		<div>{ `${ item } — a nice place to visit` }</div>
 	),
 };
+
+export const WithValidatedInput: ComponentStory< typeof FormTokenField > =
```

Comment:
Let's add an inline comment to the story too

```suggestion
/**
 * Only values for which the `__experimentalValidateInput` function returns
 * `true` will be tokenized. (This is still an experimental feature and is
 * subject to change.)
 */
export const WithValidatedInput: ComponentStory< typeof FormTokenField > =
```